roles/openshift_hosted/tasks/main.yml

```diff
@@ -1,3 +1,22 @@
 ---
+- name: Create temp directory for kubeconfig
+  command: mktemp -d /tmp/openshift-ansible-XXXXXX
+  register: mktemp
+  changed_when: False
```

Comment:
I was going to say that I can't wait until we can get rid of this, but router/registry will definitely not be covered for the first iteration of the upstream kube/openshift modules.